packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -355,6 +358,17 @@ class MediaQueryData {
   ///  * [Window.AccessibilityFeatures], where the setting originates.
   final bool boldText;
 
+  /// Describes the navigation mode requested by the platform.
+  ///
+  /// Some user interfaces are better navigated using a directional pad (DPAD)
+  /// or arrow keys, and for those interfaces, some widgets need to handle these
+  /// directional events differently. In order to know when to do that, these
```

Comment:
nit: Maybe add a one-sentence example describing what you would do differently depending on the mode. I think that would increase the understanding of what this property should be used for.

Reply:
Good idea.  I added a small paragraph about this.